Python/word-guess.py

```diff
@@ -0,0 +1,56 @@
+import random
+ 
+name = input("What's your Name? ")
+ 
+print("Good Luck ! ", name)
+ 
+words = ['rainbow', 'computer', 'science', 'programming', 
+         'python', 'mathematics', 'player', 'condition', 
+         'reverse', 'water', 'board', 'geeks','hacktoberfest'] 
+ 
+word = random.choice(words)
+ 
+ 
+print("Guess the characters")
+ 
+guesses = ''
+ 
+turns = 12
+ 
+ 
+while turns > 0:
```

Comment:
- add comments to explain the logic
- remove extra spaces from the program

Reply:
okay